patterns/jolla-hw-adaptation-hammerhead.yaml

```diff
@@ -74,6 +74,11 @@ Requires:
 # Updates after 1.1.7 will autodetect the modem presence during runtime and change
 # layout accordingly
 - jolla-settings-layout
+- voicecall-ui-jolla
```

Comment:
ok is simkit really needed?

Alin



---
Dr Alin Marin Elenahttp://alin.elena.space



---- On Sun, 13 Sep 2015 00:48:28 +0000 Simonas Leleiva &lt;notifications@github.com&gt; wrote ---- 

In patterns/jolla-hw-adaptation-hammerhead.yaml:
 &gt; @@ -74,6 +74,11 @@ Requires: &gt; # Updates after 1.1.7 will autodetect the modem presence during runtime and change &gt; # layout accordingly &gt; - jolla-settings-layout &gt; +- voicecall-ui-jolla  This should be jolla-sailfish-cellular-apps, to pull all needed bits: mms-engine
 provisioning-service
 simkit
 telepathy-ring
 voicecall-ui-jolla
 —
Reply to this email directly or view it on GitHub.